### PR TITLE
Every day's a good day when you paint

### PR DESCRIPTION
**Pull request by MadmanMartian**
https://www.youtube.com/watch?v=YLO7tCdBVrA

:cl:
* rscadd: Adds the joy of painting religion. Tags are bob ross, art, happy little trees, happy little clouds.

### CODE REVIEW

File: code/datums/religions.dm [hunk 1]
```diff
@@ -895,4 +895,18 @@
...
+	deity_name = "Bob Ross"
+	bible_names = list("The Joy of Painting")
+	male_adept = "Painter"
+	female_adept = "Painter"
+	keys = list("art", "bob ross", "happy little trees", "happy little clouds")
+
+/datum/religion/art/equip_chaplain(var/mob/living/carbon/human/H)
+	H.put_in_hands(new /obj/item/mounted/frame/painting)
+	H.h_style = "Big Afro"
+	H.f_style = "Full Beard"
```
Comment (ShiftyRail) [inline code/datums/religions.dm:911]: I think female characters don't have sprites for bread and therefore may or may not break with this.

File: code/datums/religions.dm [hunk 2]
```diff
@@ -895,4 +895,18 @@
...
 	male_adept = "Expedition Leader"
 	female_adept = "Expedition Leader"
-	keys = list("armok", "dwarf", "dorf", "dwarf fortress", "dorf fort")
\ No newline at end of file
+	keys = list("armok", "dwarf", "dorf", "dwarf fortress", "dorf fort")
+
+/datum/religion/art
+	name = "The Joy of Painting"
+	deity_name = "Bob Ross"
+	bible_names = list("The Joy of Painting")
```
Comment (ShiftyRail) [inline code/datums/religions.dm]: There's no need to use `bible_names` for a single bible, you can just set `bible_name = "The Joy of Painting"`

File: code/datums/religions.dm (hunk 1 above)
Comment (MadmanMartian) [reply code/datums/religions.dm:911]: They do. It forced the beard regardless of gender, so kept it as a feature

**Comment by DamianX** [comment]
Remember to update the wiki

**Pull request merged**